app/(dashboard)/dashboard/charts/utlis.ts

```
@@ -0,0 +1,197 @@
+  let nodes: ResourceInstanceNode[] = [];
+
+  Object.entries(detailedNetworkTopology).forEach(([, topologyDetails]) => {
+    if (topologyDetails.hasCompute) {
+      const nodesList = topologyDetails.nodes || [];
+      nodes.push(...nodesList);
+    }
+  });
+  //@ts-ignore
+  nodes = nodes.filter((node) => !node.isServerless);
+  //@ts-ignore
+  nodes = _.uniqBy(nodes, "id");
```

ESLint

[error] '_' is not defined.

(no-undef)